examples/kafka-test/glee.config.js

```diff
@@ -0,0 +1,9 @@
+export default async function () {
+  return {
+    kafkaOptions: {
+        authentication: {
+            keyFilePath: 'XIL3PSQNLE3ROP7A',
```

Comment:
Documentation says this should be `key` instead of `keyFilePath`. Let's rename this to `key` because it's clear this is not a file path.